roxie/ccd/ccdcontext.cpp

```diff
@@ -3531,6 +3525,29 @@ class CRoxieServerContext : public CRoxieContextBase, implements IRoxieServerCon
         throwUnexpected();
     }
 
+    IConstWUResult *getExternalResult(const char * wuid, const char *name, unsigned sequence)
+    {
+        Owned <IRoxieDaliHelper> daliHelper = connectToDali();
+        Owned<IConstWorkUnit> externalWU = daliHelper->attachWorkunit(wuid, NULL);
```

Comment:
Should this be checking daliHelper && daliHelper->connected()?

The code is different from getWorkunitRowReader() which I would expect it to match.